binder/reglue_highlight_dates.ipynb

```diff
@@ -0,0 +1,219 @@
+    "and provide a dictionary of new dates that we'd like to use to run the notebook.\n",
+    "\n",
+    "Let's first execute the notebook!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import papermill as pm\n",
+    "import scrapbook as sb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Ensure our outcome folder exists\n",
+    "if not os.path.exists('./outcomes'):\n",
+    "    os.mkdir('./outcomes')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_dates_one = {'start_date': \"2014-01-01\", \"stop_date\":\"2015-02-02\"}\n",
+    "pm.execute_notebook('./highlight_dates.ipynb', './outcomes/highlight_dates_run_one.ipynb', new_dates_one);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_dates_two = {'start_date': \"2014-07-01\", \"stop_date\":\"2015-10-02\"}\n",
+    "pm.execute_notebook('./highlight_dates.ipynb', './outcomes/highlight_dates_run_two.ipynb', new_dates_two);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can read in the notebook, display the metadata that was generated by scrapbook, and visualize the figure\n",
+    "we've created once more. Note that note the highlighted area has changed because we've changed the input\n",
+    "parameters!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "out_one = sb.read_notebook('./outcomes/highlight_dates_run_one.ipynb')\n",
+    "out_one.scrap_dataframe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "out_one.reglue('highlight_dates_fig')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Again, we can also explore the output notebook:\n",
+    "\n",
+    "<a href=\"highlight_dates_run_one.ipynb\"><button type=\"button\">Link to output notebook!</button></a>\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "out_two = sb.read_notebook('./outcomes/highlight_dates_run_two.ipynb')\n",
+    "out_one.scrap_dataframe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "out_two.reglue('highlight_dates_fig')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the second notebook source:\n",
+    "\n",
+    "<a href=\"highlight_dates_run_two.ipynb\"><button type=\"button\">Link to output notebook!</button></a>"
```

Comment:
Also 404. Perhaps `<a href="./outcomes/highlight_dates_run_two.ipynb"><button type="button">Link to output notebook!</button></a>`